sound/soc/sof/sof-priv.h

```diff
@@ -378,10 +378,12 @@ struct sof_ipc_fw_tracing_ops {
  * struct sof_ipc_pm_ops - IPC-specific PM ops
  * @ctx_save:		Function pointer for context save
  * @ctx_restore:	Function pointer for context restore
+ * @set_core_state:	Function pointer for turning on/off a DSP core
  */
 struct sof_ipc_pm_ops {
 	int (*ctx_save)(struct snd_sof_dev *sdev);
 	int (*ctx_restore)(struct snd_sof_dev *sdev);
+	int (*set_core_state)(struct snd_sof_dev *sdev, int core_idx, bool on);
```

Comment:
we should mark those as optional/mandatory, this can be done in a follow-up patch @ujfalusi 

in one of the last patches you set ctx_save but not ctx_restore, this is worthy of comments for the rest of us.

Reply:
I'll prepare a fixup series to to mark these optional in the struct doc.